contribs/gmf/src/floor/floorselectorComponent.js

```diff
@@ -160,8 +161,7 @@ class Controller {
  * Example:
  *
  *      <gmf-floorselector class="gmf-floorselector ol-unselectable ol-control"
- *        value="mainCtrl.dimensions.FLOOR"
- *        items="::mainCtrl.floors">
+ *        value="mainCtrl.dimensions.FLOOR">
```

Comment:
We need to know here how to configure floors list.

Reply:
+1, fixed :-)